src/cli.ts

```diff
@@ -176,12 +178,24 @@ function combineResults(results: BenchmarkResult[]): BenchmarkResult {
 }
 
 export async function main() {
-  const opts = commandLineArgs(optDefs) as Opts;
+  const opts = commandLineArgs(optDefs, {partial: true}) as Opts;
   if (opts.help) {
-    console.log(commandLineUsage([{
-      header: 'lit-benchmarks-runner',
-      optionList: optDefs,
-    }]));
+    console.log(commandLineUsage([
+      {
+        header: 'tach',
+        content: 'https://github.com/PolymerLabs/tachometer',
+      },
+      {
+        header: 'Usage',
+        content: 'tach * \n' +
+            'tach my-bench-a my-bench-b\n' +
```

Comment:
I think the confusing thing is more the things here that *aren't* URLs, since they're directories matching tachometer's particular expected file layout (<implementation>/<name>). I think it should instead be an arbitrary path to an HTML file (or a directory where index.html is assumed), which would then make this help more intuitive to show by example. That's the next thing I want to fix in tachometer very soon, so I think I'll punt on trying to make this help text more clear in the meantime.